dashboard/config/scripts_json/20-hour.script_json

```diff
@@ -0,0 +1,4134 @@
+      "absolute_position": 12,
+      "lockable": false,
+      "relative_position": 12,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "Song Writing",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "The Farmer 2",
+      "name": "The Farmer 2",
+      "absolute_position": 13,
+      "lockable": false,
+      "relative_position": 13,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "The Farmer 2",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "Abstraction",
+      "name": "Abstraction",
+      "absolute_position": 14,
+      "lockable": false,
+      "relative_position": 14,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "Abstraction",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "The Artist 4",
+      "name": "The Artist 4",
+      "absolute_position": 15,
+      "lockable": false,
+      "relative_position": 15,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "The Artist 4",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "Relay Programming",
+      "name": "Relay Programming",
+      "absolute_position": 16,
+      "lockable": false,
+      "relative_position": 16,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "Relay Programming",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "The Farmer 3",
+      "name": "The Farmer 3",
+      "absolute_position": 17,
+      "lockable": false,
+      "relative_position": 17,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "The Farmer 3",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "The Internet",
+      "name": "The Internet",
+      "absolute_position": 18,
+      "lockable": false,
+      "relative_position": 18,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "The Internet",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "The Artist 5",
+      "name": "The Artist 5",
+      "absolute_position": 19,
+      "lockable": false,
+      "relative_position": 19,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "The Artist 5",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    },
+    {
+      "key": "Wrap-up",
+      "name": "Wrap-up",
+      "absolute_position": 20,
+      "lockable": false,
+      "relative_position": 20,
+      "properties": {
+      },
+      "seeding_key": {
+        "lesson.key": "Wrap-up",
+        "lesson_group.key": "",
+        "script.name": "20-hour"
+      }
+    }
+  ],
+  "script_levels": [
```

Comment:
what determines the order of the script levels? if we change to GUIDs, are these all going to be reordered?

Reply:
The order of the objects in the json shouldn't matter here, since it's all script levels for this script. As long as the chapter and position values are correct, it's correct, right?

Reply:
right, although some of the debug-ability benefits go away if a modification to curriculum data cause things to get completely reordered.